ui/src/logic/useReactQuerySubscription.tsx

```diff
@@ -55,8 +55,7 @@ export default function useReactQuerySubscription({
   }, [app, path, queryClient, queryKey]);
 
   return useQuery(queryKey, fetchData, {
-    retryOnMount: false,
-    refetchOnMount: false,
+    staleTime: 20 * 60 * 1000,
```

Comment:
I think we probably want something much shorter. Not sure what the right default is though. Worst case scenario you're behind by a minute, but we may need to override in certain places. Let's try this for now!

```suggestion
    staleTime: 60 * 1000,
```